### ISSUE TITLE
Bummr is no longer able to rebase in version 0.4.0

### ISSUE THREAD

**Issue by monfresh** [lpender/bummr#58]
Hello. I'm not sure what is causing this, but every time I run `bummr update`, it fails to create the separate commits for each gem update, and displays the following error:
```
fatal: pathspec 'vendor/cache' did not match any files
Commit: Update yard from 0.9.16 to 0.9.18
On branch mb-bummr
Changes not staged for commit:
	modified:   Gemfile.lock

no changes added to commit
error: cannot rebase: You have unstaged changes.
error: Please commit or stash them.
Ready to run bummr.
Do you want to test the build now?
```
I tried on 2 different repos, and with bummr versions 0.3.2, 0.4.0, and the master branch. Any idea what might be going on?

I have Git version 2.20.1 installed via Homebrew.

That last time I ran bummr successfully was on November 11 with version 0.3.2. It sounds like perhaps something on my machine is causing this, possibly the Git version?

**Comment by monfresh** [comment]
Now that I think about it, I think it might possibly be related to the fish shell. It looks like I switched from zsh to fish in December. Let me try switching back to zsh and I'll report back.

**Comment by monfresh** [comment]
It fails in zsh as well.

**Comment by monfresh** [comment]
@lpender I can confirm this is a bug introduced in 0.4.0. The reason why I thought it was also reproducible in 0.3.2 is because I was running `bummr update` instead of `bundle exec bummr update`, and since I had installed newer versions, it was using those and I didn't pay attention to the running version in bummr's log.

In the same repo, if I use version 0.3.2, I have no issues, but if I use 0.4.0, then the rebase error happens.

**Comment by monfresh** [comment]
It looks like the master branch is working. Would it be possible to release a new version please?

**Comment by lpender** [comment]
Hey, just tagged/released `0.5.0`. Let me know if that works for you. Thanks!

**Comment by monfresh** [comment]
@lpender It works. Thanks!
